README.md

```diff
@@ -1,7 +1,7 @@
 # model-training
 
 [![Coverage Status](https://coveralls.io/repos/github/remla23-team06/model-training/badge.svg?branch=main)](https://coveralls.io/github/remla23-team06/model-training?branch=main)
-
+[![PRChecks](https://github.com/remla23-team06/model-training/actions/workflows/pull_requests.yml/badge.svg?branch=main)](https://github.com/remla23-team06/model-training/actions/workflows/pull_requests.yml)
```

Comment:
Currently "no status", let's see if status is updated after the pull_request workflow passes on main